lib/logger/config-serializer.ts

```diff
@@ -14,15 +14,17 @@ export default function configSerializer(
   const arrayFields = ['packageFiles', 'upgrades'];
 
   return traverse(config).map(function scrub(val: string) {
-    if (val && templateFields.includes(this.key)) {
-      this.update('[Template]');
-    }
-    if (val && contentFields.includes(this.key)) {
-      this.update('[content]');
-    }
-    // istanbul ignore if
-    if (val && arrayFields.includes(this.key)) {
-      this.update('[Array]');
+    if (this.key && val) {
```

Comment:
Why this change? Should we add a `this: xxx` type to function declaration?

Reply:
Don't know, its definition led me to traverse inner interface with `key: string | undefined`

Reply:
To be more concrete, `this` is already inferred `traverse.TraverseContext` interface with `key: string | undefined` field. We have to make sure `key` is string before searching it in `string[]` arrays.